models/_sector/dex/trades/arbitrum/_schema.yml

```
@@ -312,5 +312,24 @@ models:
             version:
               - coves1
               - coves2
+
+  - name: ramses_arbitrum_base_trades
+    meta:
```

It overwrites two versions of clippers tests.
```suggestion
              - coves1
              - coves2
              - 1
              - 2

  - name: ramses_arbitrum_base_trades
    meta:
```